Add test for selecting nodes on mac

christian-byrne opened the pull request:
Add tests from https://github.com/Comfy-Org/litegraph.js/pull/182 (drag-select on Mac)

christian-byrne committed:
Add test for selecting nodes on mac

christian-byrne commented on browser_tests/ComfyPage.ts:
```diff
@@ -914,13 +924,15 @@ export class ComfyPage {
     return new NodeReference(id, this)
   }
   async getNodeRefsByType(type: string): Promise<NodeReference[]> {
```
Assumed this is intended return type based on annotation

christian-byrne committed:
Deselect nodes in teardown

huchenlei commented:
@christian-byrne Do you have any idea why the test fails even after https://github.com/Comfy-Org/litegraph.js/pull/182 has been merged?

huchenlei commented:
> I thought it was because litegraph version wasn't updated yet

The commit should be included in v0.7.81 litegraph which was updated yesterday in https://github.com/Comfy-Org/ComfyUI_frontend/pull/1050

christian-byrne commented:
Not sure. I can fix when home 6 pst

christian-byrne committed:
Merge remote-tracking branch 'upstream/main' into mac-test

christian-byrne committed:
Fix unstable test. Remove test on unimplemented feature

christian-byrne commented:
- **Ctrl+Drag Multiple Node Selection Test**: Updated the test to draw a selection box around both clip nodes instead of just one, as the previous single-node query was unstable
- **Ctrl+Click to Add Node to Selection Test**: I initially believed this feature was already supported on Mac, but it turns out it wasn't. I submitted a PR to add this feature and moved the test case there: https://github.com/Comfy-Org/litegraph.js/pull/189
